src/main/resources/org/cbioportal/persistence/mybatisclickhouse/StudyViewFilterMapper.xml

```diff
@@ -66,6 +66,45 @@
                     '${sampleIdentifier.studyId}_${sampleIdentifier.sampleId}'
                 </foreach>
             </if>
+            <if test="studyViewFilter.customDataFilters != null and !studyViewFilter.customDataFilters.isEmpty() and customDataSamples != null">
```

Comment:
i'm not sure i understand how this customDataFilters collection (which seems to be a member of the studyViewFilter) maps back to the extracted customDataFilters which has been added as an extra parameter to the mapper methods.